apps/api/src/app/inbox/dtos/get-notifications-request.dto.ts

```diff
@@ -27,4 +27,9 @@ export class GetNotificationsRequestDto
   @IsBoolean()
   @Transform(({ value }) => value === 'true')
   archived?: boolean;
+
+  @IsOptional()
+  @IsBoolean()
+  @Transform(({ value }) => value === 'true')
```

Comment:
did you check if it works for you? 
i did something similar in https://github.com/novuhq/novu/pull/8142
but it did not work for me, and [this](https://github.com/novuhq/novu/pull/8142/files/8391ff0433f8101c90c61e927b8d99c3927bdc03#r2062566934) was the reasons

Reply:
Seems to be working fine, its coming as a string from URL query params, without the @Transform it would fail the validation